Mathlib/Geometry/Euclidean/Angle/Unoriented/UnitVectorAngles.lean

```diff
@@ -0,0 +1,171 @@
+/-
+Copyright (c) 2025 Ilmārs Cīrulis. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Ilmārs Cīrulis, Alex Meiburg
+-/
+import Mathlib.Geometry.Euclidean.Angle.Unoriented.Basic
+import Mathlib.Analysis.InnerProductSpace.Basic
+import Mathlib.Analysis.NormedSpace.Normalized
+
+/-!
+# The Triangle Inequality (for angles between unit length vectors)
+
+This file contains proof that angles between unit length vectors obey the triangle inequality.
+(Adapted from: S. W. Drury (2001), Class Notes for 189-354A (pages 12-13). https://www.math.mcgill.ca/drury/notes354.pdf, accessed 23/06/2025)
+-/
+
+namespace UnitVectorAngles
+
+open InnerProductGeometry
+
+open scoped RealInnerProductSpace
+
+variable {V : Type*}
+variable [NormedAddCommGroup V]
+variable [InnerProductSpace ℝ V]
+
+
+@[simp]
+lemma inner_self_eq_one {x : V} (hx : ‖x‖ = 1) : ⟪x, x⟫ = 1 :=
+  (inner_eq_one_iff_of_norm_one hx hx).mpr rfl
+
+lemma neg_one_le_inner {x y : V} (hx : ‖x‖ = 1) (hy : ‖y‖ = 1) : -1 ≤ ⟪x, y⟫ := by
+  have H := neg_le_of_abs_le (abs_real_inner_le_norm x y)
+  rw [hx, hy] at H
+  norm_num at H
+  exact H
+
+lemma neg_one_le_inner_normalized_normalized (x y : V) :
+    (-1 : ℝ) ≤ ⟪normalized x, normalized y⟫ := by
```

Comment:
These all seem like general lemmas that belong somewhere under `Mathlib.Analysis.InnerProductSpace`.